composer.json

```diff
@@ -4,20 +4,25 @@
     "type": "project",
     "license": "proprietary",
     "require": {
-        "spryker/console": "^3.0.0",
-        "spryker-middleware/process": "^1.0.0"
+        "ext-json": "*",
+        "spryker-middleware/process": "^1.0.0",
+        "spryker/console": "^4.0.0",
+        "spryker/queue": "^1.6",
+        "spryker/rabbit-mq": "^2.6",
+        "spryker/store": "^1.11",
```

Comment:
Please always use valid version numbers here as per our constraint guidelines for Spryker modules.